nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/ArtisticDesign.java

```diff
@@ -0,0 +1,44 @@
+package no.unit.nva.model.contexttypes;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.contexttypes.venue.Venue;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.List;
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class ArtisticDesign implements PublicationContext {
+    public static final String VENUES = "venues";
+
+    @JsonProperty(VENUES)
+    private final List<Venue> venues;
+
+    public ArtisticDesign(@JsonProperty(VENUES) List<Venue> venues) {
+        this.venues = venues;
```

Comment:
maybe we should have something like `Objects.nonNull(venues)? venues : Collections.emptyList()`?